### PR TITLE
Display interest rates in percent

### LINKED ISSUE #261: Display interest rates in percent
It was proposed to display interest rates in % as it might improve readability. We need to discuss where to change it:

#### Option A:
Change it only where it is displayed to users, backend still uses decimals:

- https://github.com/agoenergy/ptx-boa/blob/2f1306aa6f5cdb21662d780f107d40b80bc02abd/app/ptxboa_functions.py#L216 returns interest rate in [%]

- https://github.com/agoenergy/ptx-boa/blob/2f1306aa6f5cdb21662d780f107d40b80bc02abd/app/ptxboa_functions.py#L473 transforms user input interest rate in [%] back to [decimals]

- https://github.com/agoenergy/ptx-boa/blob/2f1306aa6f5cdb21662d780f107d40b80bc02abd/app/ptxboa_functions.py#L449 transforms  [decimals] to [%]  on the copy of `st.session_state["user_changes_df"]`

#### Option B:
change it throughout the app, also in the database to percent. This would involve a lot of changes

### PR DESCRIPTION
**Pull request by joAschauer**
Fixes #261

### CODE REVIEW

**Comment by markushal** [comment]
- efficiencies of electrolyzers are still per unit: 
grafik

- interest rates are in percent now, but can the number display be formatted so that a percent sign appears after the number? (e.g 5.23% instead of 5.23)
grafik

**Review by markushal — CHANGES_REQUESTED** [review]
Hi Jo, check out the comment I wrote

**Review by markushal — CHANGES_REQUESTED** [review]
Thanks! If you could now use the % format in the map legend as well, that would be perfect

grafik

**Review by markushal — CHANGES_REQUESTED** [review]
thanks! I am so sorry, but there is one other place where it still says "per unit":

grafik

File: app/user_data.py [hunk 1]
```diff
@@ -70,7 +77,7 @@ def register_user_changes(
 
         # only track the last changes if a duplicate entry is found.
         st.session_state["user_changes_df"] = pd.concat(
-            [st.session_state["user_changes_df"], res]
+            [st.session_state["user_changes_df"].astype(res.dtypes), res]
```
Comment (joAschauer) [inline app/user_data.py:80]: fixes #380

Comment (markushal) [reply app/user_data.py:80]: nice, thanks!

**Review by markushal — APPROVED** [review]

**Pull request merged**